sufsara_tutorial.Rmd

```diff
@@ -1,28 +1,54 @@
 
-Let's start with SURFsara cloud. First go to [https://ui.hpccloud.surfsara.nl/](https://ui.hpccloud.surfsara.nl/) and log in with the credentials that have been provided to you. Each student has a unique account. Then, click on your username and go to Settings. Click "Update SSH Key" and paste the public key you generated in the previous lesson into the text box, and click the "Update SSH Key" button.
+## Accessing the VM
 
-**Hint**: the easiest way to find it is by running Git GUI, though Help → Show SSH Key. In the PC lab the SSH keys are stored on the *M:* drive, so they are unique to the WUR user account. If you want to access the cloud from your own device, you will need to update the SSH key in your SURFsara account to the one on your device.
+Once you have been granted access to the *Wageningen University Geoscripting* collaboration *and* you have set up 2FA, you should be able to access your VM. Navigate back to the [starting page](https://portal.live.surfresearchcloud.nl/), and scroll down the page to the 'Workspaces' section. You should see multiple available workspaces (VMs). Please only make use of the workspace with your group name/number. If a workspace with your group name/number does not exist, please contact the staff.
 
-## Start a VM
+At the right of every workspace, there should be a (yellow) button that allows access to the workspace. Press that button for the workspace you want to access. A login-page will show up. Enter the following information:
 
-<!--### "Cloud" view-->
-You can start a virtual machine by going to the *VMs* tab and clicking the + button. Select the "Ubuntu 16.04 GeoScripting" template and press "Create" (do not change the size of the disks). Wait a bit (several minutes) until your VM enters the ready state (icon turns green). Note that the page does not update automatically: you need to click the *Refresh* (circling arrows) button in the SURFsara UI (**not** in your browser!) to update the view.
+* Use the standard Session type (Xorg)
+* Your username is your SURF researchcloud username, which is not the same as your WUR username (!). You can find your SURF username in your [profile page](https://portal.live.surfresearchcloud.nl/profile)
+* Your password is the 2FA code
 
-<!--### "User" view
-Next, press "Change view" and set it to "user". From the Dashboard, go to Virtual Resources → Templates, select the "Ubuntu 16.04 GeoScripting" template and press "Clone". Give it a name you like: this will become your very own virtual machine. A *Template* is a set of settings for your virtual machine, including the number of cores, memory and hard disks.
+If your VM is coming out of 'sleep mode', and you have to fill in a password to unlock it, you should (again) make use of your 2FA code. Happy coding!
```

Comment:
We might want to also include some information about how to log off properly (i.e. that all tasks continue running unless you use the log off button, and that you should log off before switching devices because otherwise the previous session will not get resumed: session matching is done by screen resolution)